app/models/validations/sales/sale_information_validations.rb

```diff
@@ -7,10 +7,10 @@ def validate_deposit_range(record)
     end
   end
 
-  def validate_pratical_completion_date_before_saledate(record)
-    return if record.saledate.blank? || record.hodate.blank?
+  def validate_pratical_completion_date_before_exdate(record)
+    return if record.exdate.blank? || record.hodate.blank?
 
-    unless record.saledate > record.hodate
+    unless record.exdate > record.hodate
```

Comment:
Is this part of this ticket?

Reply:
ah no, I had a pending comment addressing these, they're fixes for another PR that was already approved - seems easier just to fix them here